lab_3_ann_retriever/main.py

```diff
@@ -269,6 +378,26 @@ def retrieve_relevant_documents(
 
         In case of corrupt input arguments, None is returned.
         """
+        if (n_neighbours <= 0 or not isinstance(query, str)
+                or not query or not isinstance(n_neighbours, int)):
+            return None
+
+        query_vectorized = self._index_document(query)
+        if query_vectorized is None:
+            return None
+
+        self.index_documents(self._documents)
+        knn_list_of_tuples = self._calculate_knn(query_vectorized, self._document_vectors, n_neighbours)
+        if knn_list_of_tuples is None or not knn_list_of_tuples:
+            return None
+
+        relevant_docs = []
+        for index, float_as_value in knn_list_of_tuples:
+            if float_as_value is not None:
+                relevant_docs.append((float_as_value, self._documents[index]))
+        if relevant_docs:
```

Comment:
.

Reply:
I've added the same check for relevant_docs variable. Done, right?

Reply:
.